ganttproject/src/net/sourceforge/ganttproject/io/CSVOptionsHandler.java

```diff
@@ -1,24 +1,41 @@
-package net.sourceforge.ganttproject.io;
+/*
+Copyright 2017 Alexandr Kurutin, BarD Software s.r.o
+
+This file is part of GanttProject, an opensource project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
 
-import biz.ganttproject.impex.csv.CsvWriter;
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package net.sourceforge.ganttproject.io;
 
 /**
  * @author akurutin on 04.04.2017.
  */
 public class CSVOptionsHandler {
```

Comment:
Can't say that I understand what _Handler_ stands for -- it doesn't handle anything -- but anyway, this is going to be removed when we remove that separate options for XLS?